examples/batchmap-cat/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use numaflow::batchmap;
+use numaflow::batchmap::{BatchResponse, Datum, Message};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    batchmap::Server::new(Cat).start().await
+}
+
+struct Cat;
+
+#[tonic::async_trait]
+impl batchmap::BatchMapper for Cat {
+    async fn batchmap(&self, mut input: tokio::sync::mpsc::Receiver<Datum>) -> Vec<BatchResponse> {
+        let mut responses: Vec<BatchResponse> = Vec::new();
+             while let Some(datum) = input.recv().await {
+                let mut response = BatchResponse::from_id(datum.id);
+                response.append(Message {
+                        keys: Option::from(datum.keys),
+                        value: datum.value,
+                        tags: None,
+                });
+                responses.push(response);
+            }
```

Comment:
Do we need the indentation here? The while statement seems same level as `let mut responses`

Reply:
please run `cargo fmt --all`

Reply:
I did run that, is there some other config required for the fmt?

Reply:
You have to run it from the `examples/batchmap-cat/` directory (where `Cargo.toml` is).